src/components/RestaurantCard.js

```diff
@@ -0,0 +1,29 @@
+import { Restaurants } from './restaurants';
+import styled from 'styled-components';
+
+export default function RestaurantCard() {
+  const randomRestaurant =
+    Restaurants[Math.floor(Math.random() * Restaurants.length)];
+
+  return (
+    <SytledRestaurantCard>
+      <h2>{randomRestaurant.restaurant}</h2>
+      <p>{randomRestaurant.address}</p>
+      <StyledSpan>Description:</StyledSpan>
```

Comment:
Maybe you can find a better naming for this component? Think about the role the component takes on here.